subprojects/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/PrecompiledScriptPluginIntegrationTest.kt

```diff
@@ -14,7 +14,6 @@ class PrecompiledScriptPluginIntegrationTest : AbstractPluginIntegrationTest() {
     @Test
```

Comment:
:x: missing coverage for cacheability and relocatability of the new tasks

Reply:
actually there's already a test with some coverage, it's name should be adapted and it should cover more use cases of the precompiled script plugins

Reply:
Done in 50d135febaa